.github/workflows/release.yml

```diff
@@ -16,6 +16,16 @@ jobs:
         run: |
           bundle install
           bundle exec rake
+      - name: Increase version number
+        run: |
+          VERSION=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          sed -i "5s/.*/    VERSION = '$VERSION'/" lib/asciidoctor/chart/version.rb
+      - name: Commit & Push updated version.rb
+        run: |
+          git add .
```

Comment:
We can use the following commands: https://stackoverflow.com/a/58393457 (with a `--tags` on the `git push`).